src/main/java/com/uid2/shared/store/ScopedStoreReader.java

```diff
@@ -19,13 +19,13 @@
 public class ScopedStoreReader<T> {
     private static final Logger LOGGER = LoggerFactory.getLogger(ScopedStoreReader.class);
 
-    private final DownloadCloudStorage metadataStreamProvider;
-    private final StoreScope scope;
-    private final Parser<T> parser;
-    private final String dataTypeName;
-    private final DownloadCloudStorage contentStreamProvider;
-    private final AtomicReference<T> latestSnapshot;
-    private final AtomicLong latestEntryCount = new AtomicLong(-1L);
+    protected final DownloadCloudStorage metadataStreamProvider;
```

Comment:
Please check which are actually used by the derive class and only make those protected. Leave the other private